addons/event_sale/models/event_event.py

```diff
@@ -45,7 +45,7 @@ class Event(models.Model):
     sale_total_price = fields.Monetary(compute='_compute_sale_total_price')
     currency_id = fields.Many2one(
         'res.currency', string='Currency',
-        default=lambda self: self.env.company.currency_id.id, readonly=True)
+        compute='_compute_sale_total_price')
```

Comment:
Did you change that for a specific reason?
May I know why?

Reply:
@awa-odoo 
Yes, because if we change the currency in the settings, it will not change the currency on the event :/

Reply:
Sounds more like a bug fix then.
Did you discuss with LNA about putting the fix in an earlier stable version?

Reply:
@awa-odoo I introduced the bug in master, the bug is not present in stable version ;)
https://github.com/odoo/odoo/blob/13.0/addons/event_sale/models/event.py

Reply:
In master or saas-13.1 ?

At the very least split the change in a separate commit that explains the purpose and adds a link to the commit introducing the issue.
It's not supposed to be lost in the middle of cosmetic changes.

Reply:
@awa-odoo in master, maybe one week ago